src/Microsoft.ML.CpuMath/AvxIntrinsics.cs

```diff
@@ -15,7 +15,7 @@
 using System.Runtime.Intrinsics;
 using System.Runtime.Intrinsics.X86;
 using Microsoft.ML.Internal.CpuMath.Core;
-using nuint = System.UInt64;
+using nUInt = System.UInt64;
```

Comment:
I'd just as soon remove this type alias to avoid confusion.  Doesn't seem all that useful since it's not ifdef'ed.